config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -6,4 +6,8 @@ spec:
   label: dataplane-deployment-nova
   secrets:
     - nova-cell1-compute-config
+    # NOTE: this Secret needs to be created before deploying the control plane
```

Comment:
Did you mean dataplane? ".. created before deploying the dataplane". 

Or is something in the Control Plane using this too?